command/deployment_list_test.go

```diff
@@ -21,7 +21,7 @@ func TestDeploymentListCommand_Fails(t *testing.T) {
 	if code := cmd.Run([]string{"some", "bad", "args"}); code != 1 {
 		t.Fatalf("expected exit code 1, got: %d", code)
 	}
-	if out := ui.ErrorWriter.String(); !strings.Contains(out, cmd.Help()) {
+	if out := ui.ErrorWriter.String(); !strings.Contains(out, commandErrorText(cmd)) {
```

Comment:
In the future you can switch this style of check to use https://godoc.org/github.com/stretchr/testify/require#Contains and save a couple lines. No need to go back through and change all of these though.